public final class ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311 extends java.lang.Object implements java.lang.Runnable
{
    sortpom.exception.FailureException cap0;

    public static java.lang.Runnable bootstrap$(sortpom.exception.FailureException)
    {
        sortpom.exception.FailureException $r0;
        ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311 $r1;

        $r0 := @parameter0: sortpom.exception.FailureException;

        $r1 = new ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311;

        specialinvoke $r1.<ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311: void <init>(sortpom.exception.FailureException)>($r0);

        return $r1;
    }

    public void <init>(sortpom.exception.FailureException)
    {
        ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311 $r0;
        sortpom.exception.FailureException $r1;

        $r0 := @this: ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311;

        $r1 := @parameter0: sortpom.exception.FailureException;

        specialinvoke $r0.<java.lang.Object: void <init>()>();

        $r0.<ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311: sortpom.exception.FailureException cap0> = $r1;

        return;
    }

    public void run()
    {
        ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311 $r0;
        sortpom.exception.FailureException $r1;

        $r0 := @this: ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311;

        $r1 = $r0.<ExceptionConverterTest$lambda_failureExceptionShouldKeepCause_3__311: sortpom.exception.FailureException cap0>;

        staticinvoke <sortpom.exception.ExceptionConverterTest: void lambda$failureExceptionShouldKeepCause$3(sortpom.exception.FailureException)>($r1);

        return;
    }
}
